soc/riscv/riscv-privilege/telink_b9x/soc.c

```diff
@@ -4,11 +4,20 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 #include <sys.h>
+#elif CONFIG_SOC_RISCV_TELINK_B95
+#include <lib/include/sys.h>
+#include <zephyr/sys/printk.h>
```

Comment:
remove unused #include <zephyr/sys/printk.h>

Reply:
ok resolve this warning and after revert this change 
as well we can use one sys.h incluse for all SoCs here